network-api/networkapi/wagtailpages/README.md

```diff
@@ -74,7 +74,7 @@ The base CTA model has a title, header, description, and a newsletter identifier
 This is an empty subclass of CTA, as the base fields already cover the signup CTA case.
 
 #### 2. Petition
-
+<!-- UPDATE? -->
 This is a subclass of CTA that adds checkbox labels and Basket->Salesforce submission. See its definition in `models.py` for the full list of fields this CTA makes use of.
```

Comment:
I don't see our use of basket in the codebase...am I missing something?

Reply:
Yea... I'm that sure either. @Pomax I know for example for petition we are posting to `https://foundation.mozilla.org/api/campaign/petitions/<id>`. Is this considered as posting to Basket or Saleforce directly? 🙄

Reply:
we don't use basket directly anywhere in the codebase (anymore).

Reply:
^ That's super helpful. Thanks @Pomax